src/vs/workbench/contrib/search/browser/searchActionsFind.ts

```diff
@@ -406,6 +408,7 @@ export async function findInFilesCommand(accessor: ServicesAccessor, _args: IFin
 					updatedText = openedView.updateTextFromFindWidgetOrSelection({ allowUnselectedWord: typeof args.replace !== 'string' });
 				}
 				openedView.setSearchParameters(args);
+				openedView.toggleQueryDetails(false, typeof args.showIncludesExcludes === 'boolean')
 
```

Comment:
This would resolve as true if it was a boolean, not if it's true/false. 
I also would only want this to run if `args.showIncludesExcludes` is defined.
Also, this is missing a semicolon at the end.

Reply:
🙈 ...